### PR TITLE
Update triplea_maps.yaml

### PR DESCRIPTION
**Pull request by BeornTheBold**
Adding my map - Another World

### CODE REVIEW

File: triplea_maps.yaml [hunk 1]
```diff
@@ -434,12 +434,32 @@
...
+    <br>In addition to resources from gaining territory, use merchant ships to gain income from trade.
+    <br>
+    <br>
 - mapName: Big World 2
   mapCategory: GOOD
   url: https://github.com/triplea-maps/big_world_2/archive/master.zip
   version: 1
   img: https://raw.githubusercontent.com/triplea-maps/big_world_2/master/description/TripleA_big_world2_mini.png
-  description: |
+  description:
```
Comment (RoiEXLab) [inline triplea_maps.yaml]: This | character is needed.

File: triplea_maps.yaml [hunk 2]
```diff
@@ -434,12 +434,32 @@
     <br>Use the special tech tree system to gain access to specific units.
     <br>
     <br>
+- mapName: Another World
+  mapCategory: GOOD
+  url: https://raw.githubusercontent.com/triplea-maps/another_world/master.zip
+  version: 3.7
```
Comment (ron-murhammer) [inline triplea_maps.yaml]: Version should just be 1. This isn't the map version. Just used to increment to indicate to the engine that there is a new download.

**Comment by RoiEXLab** [comment]
@BeornTheBold The link to the map thumbnail is invalid, the thumbnail doesn't exist.
Are you planning to add a thumbnail soon? It would be displayed in the map downloading window and on [this site](http://triplea-game.org/map/another-world/). Otherwhise I'll remove the link entry because no image is better than a "missing image" image

**Comment by ron-murhammer** [comment]
@RoiEXLab Looks like just the wrong path: https://raw.githubusercontent.com/triplea-maps/another_world/master/map/map_with_sea_zone_names.png

**Comment by ron-murhammer** [comment]
@RoiEXLab https://github.com/triplea-game/triplea/pull/3087

**Comment by BeornTheBold** [comment]
Ok, I tried editing triplea_maps.yaml again to use a different path.  As you can probably see, I don’t really know how to do all of this.

- Byron

**Comment by ron-murhammer** [comment]
@BeornTheBold Yeah, first time around is a bit tough. Its fixed now. Should appear properly in the download list with the map image now. Nice work! Now ya just gotta find lots of players to try it out!

**Pull request merged**